drizzlepac/haputils/photometry_tools.py

```diff
@@ -235,9 +237,13 @@ def convert_flux_to_abmag(in_flux, photflam, photplam):
         f_lambda = in_flux * photflam
 
         # Convert f_lambda to STMAG
-        stmag = -2.5 * np.log10(f_lambda) - 21.10
+        # ignores warnings from log10 of zeros and negative values
+        with np.errstate(divide='ignore', invalid='ignore'):
+            stmag = -2.5 * np.log10(f_lambda) - 21.10        
 
         # Convert STMAG to ABMAG
-        abmag = stmag - 5.0 * np.log10(photplam) + 18.6921
+        # ignores warnings from log10 of zeros and negative values
+        with np.errstate(divide='ignore', invalid='ignore'):
+            abmag = stmag - 5.0 * np.log10(photplam) + 18.6921
```

Comment:
Lines 241 and 246 introduce benign changes which eliminate lots of messages in the logs without changing the current computations.  In this sense, the changes are fine.  However, the Jira ticket was meant to _investigate how these computations can be better handled_ starting from the routine which calls compute_flux_to_abmag.  Should the block of code starting on Line 231 be expanded to encompass handling the flux array?  If issues are handled in the initial block of code we never drop down into the log10 computations.

In the test dataset was photplam zero or negative?  It is a pivot wavelength, so I am curious.